Editor/AGS.Types/Room.cs

```diff
@@ -379,7 +379,22 @@ public Interactions Interactions
             get { return _interactions; }
         }
 
-		void IChangeNotification.ItemModified()
+        public double GetMaskScale(RoomAreaMaskType mask)
+        {
+            switch (mask)
+            {
+                case RoomAreaMaskType.WalkBehinds:
+                    return 1.0; // walk-behinds always 1:1 with room size
+                case RoomAreaMaskType.Hotspots:
+                case RoomAreaMaskType.WalkableAreas:
+                case RoomAreaMaskType.Regions:
+                    return 1.0 / MaskResolution;
+                default:
+                    return 0.0;
```

Comment:
Should not this throw exception instead of returning 0.0? As this happens in case of unknown mask type was passed.

Reply:
Yes probably, I was copying from `RoomStruct::GetMaskScale(RoomAreaMask mask)`, but I'll rewrite it